src/pageLoad.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable*/
+import nav from "./nav";
+import "./css/styles.css";
+import homeTab from "./homeTabContent";
+
+const contentDiv = document.getElementById("content");
+
+const tab = () => {
+  const tab = document.createElement("div");
+  tab.id = "tab";
+  tab.classList.add("tab", "flex");
+  tab.appendChild(homeTab());
+  return tab;
+};
+
+const pageLoad = () => {
+  contentDiv.classList.add("container");
+  contentDiv.appendChild(nav());
+  contentDiv.appendChild(tab());
+};
+
+export default pageLoad;
+/* eslint-disable*/
```

Comment:
- **_Fix the lint error(or disable an error, avoid overuse of linter disable) instead of disabling eslint on the whole file_**